plugin/trino-singlestore/src/main/java/io/trino/plugin/singlestore/SingleStoreClient.java

```diff
@@ -721,7 +720,7 @@ private static LongReadFunction singleStoreTimeReadFunction(TimeType timeType)
                 }
                 return rounded;
             }
-            catch (DateTimeParseException e) {
+            catch (RuntimeException e) {
```

Comment:
This change seems wrong or insufficient. We should override `isNull` method in my understanindg:
```
	at io.trino.plugin.jdbc.ReadFunction.isNull(ReadFunction.java:29)
	at io.trino.plugin.jdbc.JdbcPageSource.getNextSourcePage(JdbcPageSource.java:193)
```